configuration/concepts/disposition.csv

```diff
@@ -4,5 +4,5 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:f
 89cd4d9f-579a-4ee5-9353-f62704688ea0,,org.openmrs.module.emrapi:DISCHARGE,Discharge Patient,Patient de sortie,Discharge Patient,Patient de sortie,,,Misc,N/A,,,,
 a821d9a6-5486-40f9-b738-9180bff73d83,,org.openmrs.module.emrapi:UNDO_DISCHARGE,Cancel Patient Discharge,Annuler le congé du patient,Cancel Patient Discharge,Annuler le congé du patient,,,Misc,N/A,,,,
 d664e628-2957-4b02-8a25-28a8cb10fee5,,org.openmrs.module.emrapi:Disposition,Disposition,Disposition,Disposition,Disposition,,,Question,Coded,Référer le patient;Admettre le patient;Patient de sortie;Annuler le congé du patient,,,
-a6ea10ed-ede7-43e2-83e8-c6be5ca26d76,,org.openmrs.module.emrapi:DispositionNote,Disposition Note,Note de disposition,Disposition Note,Note de disposition,,,Misc,Text,,,,
-caa44025-9e30-4feb-8d2a-c58c91f36bfc,,org.openmrs.module.emrapi:Disposition Concept Set,Disposition Set,Ensemble de disposition,Disposition Set,Ensemble de disposition,,,Misc,N/A,,Disposition;Note de disposition,,
\ No newline at end of file
+a6ea10ed-ede7-43e2-83e8-c6be5ca26d76,,org.openmrs.module.emrapi:DispositionNote,Disposition Note,Disposition Note,Disposition Note,Disposition Note,,,Misc,Text,,,,
+caa44025-9e30-4feb-8d2a-c58c91f36bfc,,org.openmrs.module.emrapi:Disposition Concept Set,Disposition Set,Disposition Set,Disposition Set,Disposition Set,,,Misc,N/A,,Disposition;Note de disposition,,
```

Comment:
The Disposition Concept Set will fail because the name you added is still french...change the name to the expected one of `Disposition Note`